astropy/table/tests/test_operations.py

```diff
@@ -2424,3 +2424,63 @@ def test_mixin_join_regression():
     t12 = table.join(t1, t2, keys=("index", "flux1", "flux2"), join_type="outer")
 
     assert len(t12) == 6
+
+
+@pytest.mark.parametrize(
```

Comment:
I wonder if the test would be clearer if it gave the result in the parametrization rather than do a check to see what would be expected. I.e., "t1, t2, eq"

Reply:
I asked myself the same question as I wrote the test, but I found that the `try/except` pattern expressed the documented behaviour more clearly than what *could* feel like a list of ad-hoc expectations. Honestly it seems arguable both ways. Maybe @taldcroft has stronger opinions that could help forming a decision ?

Reply:
Also reasonable. As said, I am absolutely fine with just merging the PR as is! (in another PR, @taldcroft pointed out the old adage of perfect being the enemy of good enough)

Reply:
When I looked at this I had the same idea of "t1, t2, eq" being a little more clear, but then I reminded myself of perfect and good. I think this is quite good. :smile:

Reply:
Having said that @neutrinoceros - I generally agree with @mhvk's sentiment so that is something to keep in mind going forward.

Reply:
I'll try !